verifier/trustpolicy/trustpolicy.go

```diff
@@ -289,7 +289,7 @@ func LoadDocument() (*Document, error) {
 // SignatureVerification struct throws error if SignatureVerification is invalid
 func (signatureVerification *SignatureVerification) GetVerificationLevel() (*VerificationLevel, error) {
 	if signatureVerification.VerificationLevel == "" {
-		return nil, errors.New("trust policy statement is missing or has empty signature verification level, it must be specified")
+		return nil, errors.New("signature verification configuration is missing or contains empty verification level, it must be specified")
```

Comment:
This is first time I am seeing `signature verification configuration`, why do we want to move away from trust policy statement?

Reply:
The name comes from the corresponding field called `signatureVerification` in the [schema spec](https://github.com/notaryproject/notaryproject/blob/v1.0.0-rc.2/specs/trust-store-trust-policy.md#trust-policy-schema)

If we want to be accurate, how about changing the error message to `verification level cannot be empty`

Reply:
Why this change is needed: the validation error comes from the verification level, the original msg might make notation user think the statement itself is missing.